build.sbt

```diff
@@ -243,3 +245,21 @@ def runStreamExampleWithOtelAgent = Command.command("runStreamExampleWithOtelAge
     Project.extract(newState).runInputTask(Compile / runMain, " example.SimpleStreamExample", newState)
   s
 }
+
+def runZioExampleWithOtelAgent = Command.command("runZioExampleWithOtelAgent") { state =>
+  val extracted = Project extract state
+  val newState = extracted.appendWithSession(
+    Seq(
+      run / javaOptions ++= Seq(
+        s"-javaagent:$projectRootDir/opentelemetry-javaagent-$OpentelemetryLatestVersion.jar",
+        s"-Dotel.service.name=mesmer-stream-example",
```

Comment:
incorrect name

Reply:
@lgajowy, refactored in this commit:
https://github.com/ScalaConsultants/mesmer/pull/470/commits